### DISCUSSION TITLE
Markers not displaying on @react-google-maps/api after updating to latest version of Next.js

### DISCUSSION THREAD

**Discussion by gundogduyakici** [JustFly1984/react-google-maps-api#3294] [General]
**Summary**
I recently updated my project to use the latest version of Next.js. After the update, I noticed that the markers on my Google Maps (implemented using @react-google-maps/api) are no longer displaying. However, when I revert back to the older version of Next.js, the markers appear as expected.

**Steps to Reproduce:**

1. Update to the latest version of Next.js.
2. Use @react-google-maps/api to implement Google Maps with markers.
3. Notice that the markers are not displaying.

**Expected Behavior:**
Markers should display on the map as they do when using the older version of Next.js.

**Actual Behavior:**
Markers are not displaying on the map after updating to the latest version of Next.js.

**Additional information**
```
Next.js version: 13.5.3
@react-google-maps/api version: 2.19.2
Node.js version: 18.14.0
```

**Comment by ChinonsoIg** [comment]
Were you able to find solution to this? @gundogduyakici 
I have the same issssue now

**Reply by gundogduyakici** [reply]
Hi Chinonsolg,
I updated to the latest version of Next.js and the problem is fixed. The problem was not with this package but with the next.js version. With the latest update of next.js, this problem is now solved.

**Reply by ChinonsoIg** [reply]
Alright, thank you. I'll update my Next app.
